devel/ansible/roles/dev/files/securitas.cfg

```diff
@@ -0,0 +1,22 @@
+TEMPLATES_AUTO_RELOAD = True
```

Comment:
Is this whole file worth duplicating? Can it use the one from the root of the repo instead?

Reply:
okies, tweaked this, removing the duplicated file, and changed the default cfg a bit to suit, since these appeared to be dummy values anyways.

also, changed the password we add to the ipa server in vagrant to match what was already in the default cfg.